### DISCUSSION TITLE
Job parallelism for MD with i-PI

### DISCUSSION THREAD

**Discussion by mcourtiel** [deepmodeling/deepmd-kit#3129] [Q&A]
Hello everyone,

I am writing to ask for more precision about the parallelism of a job when I want to run MD (or PIMD) with I-Pi.

I have read the 11.2 section of the documentation ("How to control the parallelism of a job?") but I might need a bit of help. I read that we can use this command lines in order to parallelize the training of a model : 
export OMP_NUM_THREADS
export TF_INTRA_OP_PARALLELISM_THREADS
export TF_INTER_OP_PARALLELISM_THREADS
And this is working well,  but I am wondering how it works when it comes to using the DeepMD potential when I want to perform molecular dynamic with I-PI.

Thanks a lot !

**Comment by njzjz** [comment]
Do you mean you found they did not work for i-PI?
